tasks/main.yml

```diff
@@ -26,6 +26,6 @@
     - Restart Docker
 
 - name: Install Docker API client for Python
-  pip: name=docker-py
+  pip: name=docker
```

Comment:
Since this is going to break backwards compatibility anyway (major revision bump), I think that it makes sense to rename `docker_py_version` to something like `docker_python_sdk_version` or something like that.